gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest_asyncio.py.j2

```diff
@@ -0,0 +1,5 @@
+{# TODO: Remove the following condition for async rest transport once support for it is GA:
+ # {% if rest_async_io_enabled %}
```

Comment:
nit: It could be confusing to have commented out jinja code. Could we remove `# {% if rest_async_io_enabled %}`?

Reply:
See related comment: https://github.com/googleapis/gapic-generator-python/pull/2123#discussion_r1742785359

Reply:
Will be addressed in https://github.com/googleapis/gapic-generator-python/pull/2123

Reply:
Addressed in https://github.com/googleapis/gapic-generator-python/pull/2123